src/Microsoft.IdentityModel.Tokens/Validation/ValidationParameters.cs

```diff
@@ -499,7 +499,7 @@ public TokenReplayValidationDelegate TokenReplayValidator
         /// </remarks>
         /// <exception cref="ArgumentNullException">Thrown when the value is set as null.</exception>
         /// <returns>The <see cref="TokenTypeValidationDelegate"/> used to validate the token type of a token</returns>
-        public TokenTypeValidationDelegate TypeValidator
+        public TokenTypeValidationDelegate TokenTypeValidator
```

Comment:
Is this a breaking change?

Reply:
The type `ValidationParameters` is internal and not used anywhere outside of our new `ValidateTokenAsync`.
So I would say not a public interface change, even if the property is marked as public.

Reply:
That's right, thanks Ignacio. Resolving this thread!